### PR TITLE
Manually bump CDI to v1.57.0-rc1

### PR DESCRIPTION
**Pull request by akalenyu**
We want to start testing this with kubevirt, but the script only picks up "latest" stable releases.

### CODE REVIEW

File: cluster-provision/k8s/fetch-latest-cdi.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ fi
 [ ! -d "$provision_dir" ] && echo "directory $provision_dir does not exist!" && exit 1
 [ ! -d "$provision_dir/manifests" ] && echo "directory $provision_dir/manifests does not exist!" && exit 1
 
-tag_name=$(curl -s -L -f https://api.github.com/repos/kubevirt/containerized-data-importer/releases/latest | jq -r '.tag_name')
+tag_name=$(curl -s -L -f https://api.github.com/repos/kubevirt/containerized-data-importer/releases/111644568 | jq -r '.tag_name')
```
Comment (brianmcarey) [inline cluster-provision/k8s/fetch-latest-cdi.sh:42]: I think this should remain as latest ? since the script is called `fetch-latest-cdi.sh`

**Review by brianmcarey — COMMENTED** [review]
The bump to the rc version looks good but I think `fetch-latest-cdi.sh` should remain unchanged.

File: cluster-provision/k8s/fetch-latest-cdi.sh (hunk 1 above)
Comment (akalenyu) [reply cluster-provision/k8s/fetch-latest-cdi.sh:42]: whoops my bad

**Review by brianmcarey — APPROVED** [review]
/approve

**Pull request merged**